.eslintrc.js

```diff
@@ -320,8 +320,9 @@ module.exports = {
           jsx: true,
         },
       },
-      plugins: ['react'],
+      plugins: ['react', 'react-compiler'],
```

Comment:
Is react-compiler the right plugin? https://react.dev/blog/2025/10/07/react-compiler-1 says

> Compiler-powered lint rules ship in eslint-plugin-react-hooks’s recommended and recommended-latest preset.

and 

> If you have already installed eslint-plugin-react-compiler, you can now remove it and use eslint-plugin-react-hooks@latest. Many thanks to [@michaelfaith](https://bsky.app/profile/michael.faith) for contributing to this improvement!